src/styles/sidebar/_utilities.scss

```diff
@@ -1,3 +1,33 @@
-@use '@hypothesis/frontend-shared/styles/util';
-
 @use 'tailwindcss/utilities';
+
+// This is a temporary local implementation of `.hyp-u-outline-on-keyboard-focus`
+// and `.hyp-u-outline-on-keyboard-focus--inset`. These are the last remaining
+// utilities used by this application, so implementing them here allows us to
+// remove the `frontend-shared` utility styles from the `sidebar` CSS.
+// TODO: This should be re-implemented as a Tailwind plugin in `frontend-shared`
```

Comment:
What might the Tailwind version of this look like?

Reply:
I could push a working branch to `frontend-shared` if you'd like to see.